lib/bolt/cli.rb

```diff
@@ -222,6 +213,46 @@ def create_option_parser(results)
           raise Bolt::CLIExit
         end
       end
+
+      # add relivant options to parser for given mode
+      nodes_args = [
```

Comment:
I think we should continue to put this first in the parser options (when used).

Reply:
Fixed